sshcode.go

```diff
@@ -477,7 +489,8 @@ func syncExtensions(sshFlags string, host string, back bool) error {
 		return err
 	}
 
-	const remoteExtensionsDir = "~/.local/share/code-server/extensions/"
+	//const remoteExtensionsDir = ".local/share/code-server/extensions/"
+	var remoteExtensionsDir = windowsVarFix("remoteExtensionsDir")
```

Comment:
This should just be
```go
remoteExtensionsDir := "~/.local/share/code-server/extensions/`
if runtime.GOOS == "windows" {
  remoteExtensionsDir = "blah"
}
```

Reply:
Wait. an `if` statement works outside of a function? I had to do it the way i did just to avoid the annoying thing called "Var Shadowing", and it works.

Reply:
if statements don't work outside of a function, this code is inside of a function.

I'm saying replace these two lines with what I said and remove the `windowsVarFix` function.

Var shadowing doesn't apply here since we're not declaring a new variable. We'd run into an issue if the line in the if statement was `:=` instead of `=`.

Reply:
I have actually made a major rewrite to windowsVarFix, that basically does this, go to https://cdn.merith.tk/ and navigate to Workspace/sshcode and see the stuff there, that is my current actual workspace, read only, and look at the new function, I am slowly working on adding your edits without loosing functionality, but have encountered a issue where it's not playing nice with pathing, (major issues that are practically game breaking, will need to debug and fix before I can continue at all)

Reply:
I'm saying that the windowsVarFix function is unnecessary because it's only used in two places and performs a small task. It should be removed in favor of my suggestion

Reply:
I see what your saying, and i will listen and apply it, but what about future proofing? what if more features are implemented that need it for windows?